### DISCUSSION TITLE
Welcome to KelloggBot Discussions!

### DISCUSSION THREAD

**Discussion by SeanDaBlack** [SeanDaBlack/KelloggBot#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KidPluto** [comment]
I am just watching in real time as people contribute improvements.  Very enjoyable.  And all towards an important goal, of supporting the striking workers.

**Reply by pws1453** [reply]
Lol, I contributed a little, but it quiiickly fell out of the scope of what I currently know - it's been an absolute pleasure to see how everyone is contributing.

**Reply by BigweldIndustries** [reply]
i finally feel like i can apply my miscellanious python skills lol, hope I can contribute even more

**Comment by ghost** [comment]
Hey all! Amazing work here <3
I'm not super familiar with python so I ported the repo to javascript for anyone interested: https://github.com/danithecyberpunkgirl/applybot

I've got a slightly different captcha solution than you guys but I doubt it matters much. I'm planning to target some of the temp agencies Kellogg's is trying to use with my version of the bot in the coming days.

**Reply by ghost** [reply]
lmao ya'll are outpacing me very quickly. Gonna abandon my project and start helping here even though I'm not super familiar with python
